docs/web-ui-auto-update-issue.md

```diff
@@ -0,0 +1,24 @@
+# Web UI Auto-Update Issue
+There is a known issue for old installation users(v1.0), the UI will get updated automatically after restarting the converter service. Since v2.0 is not compatible with v1.0, the old installation users may have some problems using the updated UI. 
+
+# Cause
+This issue is caused by the ```latest``` tag of the docker image in the deployment template file. After v2.0 is released, the ```latest``` image is updated to the v2.0 version that supports C-CDA files and is compatible with the previous versions. When you restart the converter service, it will stop the current container and pull the updated docker image to restart. As a result, the UI of old deployments is updated to new UI automatically.
+
+# Solution
+If you encounter this issue, the solution is to rollback the container image. Since all the templates data are stored in a [persistent shared storage](https://docs.microsoft.com/en-us/azure/app-service/containers/configure-custom-container#use-persistent-shared-storage), the container image rollback will not lose your templates.
```

Comment:
Did you mean to say that "Rolling back the container image will restore your previous templates"?